packages/pancake-uikit/src/components/Footer/Footer.tsx

```diff
@@ -50,6 +50,7 @@ const MenuItem: React.FC<FooterProps> = ({
                   <Link
                     href={href}
                     target="_black"
```

Comment:
Forgive my ignorance, is _black a valid html target?

Reply:
This a typo... well this explains the behaviour you had on links. `rel="noopener"` is actually not needed with `_blank` as target. Will fix the typo instead, thanks.

Reply:
Ha! I was searching google and was like man I must have been out of the react and html game for too long. Glad the issue had a root cause.